packages/remix-cloudflare-pages/magicExports/platform.ts

```diff
@@ -0,0 +1,4 @@
+// This file lists all exports from this package that are available to `import
+// "remix"`.
+
+export { createCloudflareKVSessionStorage } from "@remix-run/cloudflare-workers";
```

Comment:
Think this should be `@remix-run/cloudflare-pages` if you're copying that over to this package as well.